tables/tr.ctb

```diff
@@ -20,26 +22,46 @@
 # BRLTTY Text Table - Turkish
 #
 # Samuel Thibault <samuel.thibault@ens-lyon.org>
-# 
+#
 # This table is based on the Unesco report on the progress of unification of
 # braille writing « L'ÉCRITURE BRAILLE DANS LE MONDE », by Sir Clutha
 # MACKENZIE: http://unesdoc.unesco.org/images/0013/001352/135251fo.pdf
 # The document is dated 1954, so this table may be quite outdated.
 
-# generated by ttbtest
+# With corrections and additions by Oğuz Uğur <oguzugur2834@gmail.com>
+#
+# Definitions of punctuation symbols are based on the Turkish Braille
+# Writing Guideline released by Turkish Education Ministery in
+# 1991. For an unaccessible PDF version see
+# <https://orgm.meb.gov.tr/alt_sayfalar/yayimlar/ozelegitim/braille/braille.pdf>.
+#
+# Definitions of accented letters do not follow the guideline, but
+# were taken from de-accents.cti instead. This was done to prevent
+# mixing accented letters with contractions. For example, some
+# accented e letters should normally be represented by dot 26 or 1246,
+# but these patterns are also used for contractions.
```

Comment:
There are no contractions in computer braille, so what could be mixed up?

Reply:
This is a good question, @oguzugur28, please answer Bert question because I am not known turkish language.

Reply:
Not possible Oguz simple wanted wrote braille word part?
Usual computer braille real not have contraction, @bertfrees, you wrote right, but unfortunately @oguzugur28 doesn't answered this question, and me not have other informations this question related.

Reply:
I've added a note in the table about this question, which I would like to be cleared up at some point, but it shouldn't block the PR.